downloads/metadata/public-financing-fields.csv

```diff
@@ -0,0 +1,4 @@
+---
+---
+Field Name,Question,Short Label,Additional Notes{% for control in site.data.sections.["public-financing"].controls %}
```

Comment:
Not that I can see.

I just tried removing the period and that got rid of the warning.

`sections.["public-financing"]` Warning
`sections["public-financing"]` No warning